### PR TITLE
Adding Connectome Examples

### PR DESCRIPTION
**Pull request by johnisanerd**
Adding in the Connectome Examples, for the worm brain example for the GoPiGo, updated for the GPG3.

### CODE REVIEW

**Comment by johnisanerd** [comment]
Here's a link to the current project:  https://studio.dexterindustries.com/cwists/preview/414x

File: Software/Python/Examples/Connectome/connectome.py [hunk 1]
```diff
@@ -0,0 +1,5005 @@
...
+        postSynaptic['MDR12'][nextState] += 5
+        postSynaptic['VD3'][nextState] += 25
+        postSynaptic['VD4'][nextState] += 6
+
+def DA4():
+        postSynaptic['AVAL'][nextState] += 3
+        postSynaptic['AVAR'][nextState] += 2
+        postSynaptic['DA1'][nextState] += 1
+        postSynaptic['DA3'][nextState] += 1
+        postSynaptic['DB3'][
```
Comment (CleoQc) [inline Software/Python/Examples/Connectome/connectome.py]: Does 27 even make sense? it's now 27 mm.  us_dist was returning values in cm, if memory serves me right.

File: Software/Python/Examples/Connectome/disembodiedConnectome.py [hunk 2]
```diff
@@ -0,0 +1,4964 @@
...
+        postSynaptic['VD4'][nextState] += 12
+        postSynaptic['VD5'][nextState] += 15
+
+def DA5():
+        postSynaptic['AS6'][nextState] += 2
+        postSynaptic['AVAL'][nextState] += 1
+        postSynaptic['AVAR'][nextState] += 5
+        postSynaptic['DB4'][nextState] += 1
+        postSynaptic['MDL13'][nextState] += 5
+        postSynaptic['MDL14'][nextState] +=
```
Comment (CleoQc) [inline Software/Python/Examples/Connectome/disembodiedConnectome.py]: either remove the commented out section, or bring it to GPG3 specs. I don't care which one.

File: Software/Python/Examples/Connectome/disembodiedEO.py [hunk 3]
```diff
@@ -0,0 +1,4959 @@
...
+        incrementNeuron('RMDDR',1)
+        incrementNeuron('RMDVL',4)
+        incrementNeuron('RMEV',1)
+        incrementNeuron('URYDL',1)
+
+def IL1DR():
+        incrementNeuron('IL1DL',1)
+        incrementNeuron('IL1R',1)
+        incrementNeuron('MDR01',4)
+        incremen
```
Comment (CleoQc) [inline Software/Python/Examples/Connectome/disembodiedEO.py]: remove commented out code

**Pull request closed without merging**